ui/app/templates/vault/cluster/metrics/index.hbs

```diff
@@ -6,8 +6,79 @@
   </p.levelLeft>
 </PageHeader>
 
-{{!-- TODO: PLACEHOLDER, replace with new component --}}
+{{!-- ARG TODO: Add permission conditionals for menu dropdowns --}}
```

Comment:
nice 👍 we definitely wouldn't want people to see the 'create' buttons if they don't have permission